devicon.json

```diff
@@ -8092,6 +8092,24 @@
         "color": "#68217A",
         "aliases": []
     },
+    {
+        "name": "vitejs",
```

Comment:
```suggestion
        "name": "vitejs",
        "altnames": [
            "vite.js",
            "Vite"
        ]
```

Reply:
I could apply the suggestion, just, could you put back the closing square bracket ?

Reply:
It's already there, just github's visuals that's acting weird for some reason.

This is what I added
```json
        "name": "vitejs",
        "altnames": [
            "vite.js",
            "Vite"
        ],
```

Reply:
I had forgotten the comma though, so added that :)

Reply:
Ok, applying suggestions then :)